documents/moving-across-committees.md

```diff
@@ -16,18 +15,16 @@ Here's a checklist for you to get started with transferring committees. This lis
-* **Step 2**: Create an issue in the [Community Committee](https://github.com/nodejs/community-committee/issues) repository asking if the Community Committee would be willing to take the Working Group in question on as a Community Committee Working Group.
-  * **Step 2.1**: If there _are no objections_, continue on to step two.
-  * **Step 2.2**: If there _are objections_ by the committee, the group requesting the transfer can either address feedback and try again or remain under their current committee.
-* **Step 3**: Once approved by both the WG and the Community Committee, you will need to create two pull requests: one pull request to be dechartered by the TSC and one pull request to be chartered by the Community Committee.
-* **Step 4**: Once dechartered by the TSC and chartered by the Community Committee, the process is complete.
+### Moving from one Committee to another Committee
 
-### Moving from the Community Committee to the TSC
-* **Step 1**: Create an issue in the repository for the Working Group in question, proposing the move. Given the governance model, this will likely be a vote and will need majority approval from the Working Group's members.
-* **Step 2**: Create an issue in the [TSC](https://github.com/nodejs/TSC/issues) repository asking if the TSC would be willing to take the Working Group in question on as a TSC Working Group.
-  * **Step 2.1**: If there _are no objections_, continue on to step two.
+* **Step 1**: Create an issue in the repository for the Working Group in question and gain approval from the Working Group for the move.
+  * **Step 1.1**:If there _are no objections_, continue on to Step 2.
+* **Step 2**: Create an issue in the repository of the new Committee asking if it would be willing to take the Working Group in question on as a Working Group of that Committee.
+  * **Step 2.1**: If there _are no objections_, continue on to step 2.2.
   * **Step 2.2**: If there _are objections_ by the committee, the group requesting the transfer can either address feedback and try again or remain under their current committee.
-* **Step 3**: Once approved by both the WG and the TSC, you will need to create two pull requests: one pull request to be dechartered by the Community Committee and one pull request to be chartered by the TSC.
-* **Step 4**: Once dechartered by the Community Committee and chartered by the TSC, the process is complete.
+* **Step 3**: Once approved by both the WG and the new Committee, you will need to create two pull requests: one pull request to be dechartered by the previous Committee and one pull request to be chartered by the new Committee.
+* **Step 4**: Once dechartered by the previous Committee and chartered by the new Committee, you can use the following checklist 
+  * [] Have the currrent committee vote to decharter
+  * [] Have the new committee vote (and approve) a new charter
```

Comment:
The "(and approve)" bit is superfluous IMO. A vote intrinsically means approving or dissaproving of a thing following an ordered process. I would remove it.